src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -3342,3 +3343,27 @@ class DescribeStatementSegment(BaseSegment):
             ),
         ),
     )
+
+
+@snowflake_dialect.segment()
+class CallStatementSegment(BaseSegment):
+    """`CALL` statement.
+
+    https://docs.snowflake.com/en/sql-reference/sql/call.html
+    """
+
+    type = "call_statement"
+    match_grammar = Sequence(
+        "CALL",
+        Sequence(
+            Ref("FunctionNameSegment"),
+            Bracketed(
+                Ref(
+                    "FunctionContentsGrammar",
+                    # The brackets might be empty for some functions...
```

Comment:
Should we add a test case for this then?

Reply:
Added the unit test 👍